### ISSUE TITLE
Failed Maven build for gwt-maven-archetypes

### ISSUE THREAD

**Issue by laurentd75** [tbroyer/gwt-maven-archetypes#33]
Hi,
I just cloned your gwt-maven-archetypes Git repo and tried to build them ("mvn clean install") using JDK 1.7u65 and Maven 3.2.5, and got the following build error:

Here's the relevant excerpt of the Maven output:

_____________________________________

[INFO] project created from Archetype in dir: C:\Local\GIT_REPOS\ThomasBroyer\gwt-maven-archetypes\modular-webapp\target\test-classes\projects\basic-webapp\project\basic-webapp
[INFO] Comparing generated project with reference content: C:\Local\GIT_REPOS\ThomasBroyer\gwt-maven-archetypes\modular-webapp\target\test-classes\projects\basic-webapp\reference
[WARNING] Contents of file pom.xml are not equal
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] gwt-maven-archetypes ............................... SUCCESS [  3.981 s]
[INFO] modular-webapp ..................................... FAILURE [  2.871 s]
[INFO] modular-requestfactory ............................. SKIPPED
[INFO] guice-rf-activities ................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 7.508 s
[INFO] Finished at: 2015-02-05T15:24:19+01:00
[INFO] Final Memory: 12M/30M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-archetype-plugin:2.2:integration-test (default-integration-test) on pr
oject modular-webapp:
[ERROR] Archetype IT 'basic-webapp' failed: Some content are not equals
[ERROR] -> [Help 1]

_____________________________________

Can you please look into this? 

FYI I tried this on vaious combinations Maven (3.0.5, 3.2.5) and JDK (1.7u65, 1.8u11), the problem must lie with the POM file in the archetype I suppose.

Hope you can help,
Thanks,
Laurent

**Comment by tbroyer** [comment]
Dupe of #22.

TL;DR: Maven and Java are OK, try changing your OS (or configuring Git, or just skipping tests).
